### PR TITLE
Gradle files cleanup and version bump

### PR DESCRIPTION
**Pull request by AnatolyPopov**

### CODE REVIEW

**Comment by AnatolyPopov** [comment]
@jeqo Fixed + fixed one more place in Dockerfile

File: docker/kafka/Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN git checkout ${AK_BRANCH}
 
 ARG SCALA_VERSION=2.12
 
-RUN ./gradlew -PscalaVersion=${SCALA_VERSION} testJar releaseTarGz
+RUN ./gradlew -PscalaVersion=${SCALA_VERSION} testJar distTar
```
Comment (jeqo) [inline docker/kafka/Dockerfile:22]: I think this one is not required as it's building upstream kafka, not the plugin

Comment (jeqo) [reply docker/kafka/Dockerfile:22]: We will get rid of this file anyway on #262

Comment (AnatolyPopov) [reply docker/kafka/Dockerfile:22]: Ah, well, you right, was doing just grep + replace

Comment (AnatolyPopov) [reply docker/kafka/Dockerfile]: Fixed

File: docker/kafka/Dockerfile [hunk 2]
```diff
@@ -37,4 +37,4 @@ COPY --from=base /kafka/storage/api/build/libs/kafka-storage-api-${KAFKA_VERSION
 ENV KAFKA_HOME /kafka
 WORKDIR ${KAFKA_HOME}
 ENV PATH $PATH:${KAFKA_HOME}/bin
-CMD [ "kafka-server-start.sh" ]
\ No newline at end of file
+CMD [ "kafka-server-start.sh" ]
```
Comment (jeqo) [inline docker/kafka/Dockerfile]: nit: just wondering if this is intentional or sometimes github get's confused with new lines at EOF.

Comment (AnatolyPopov) [reply docker/kafka/Dockerfile]: Not intentional but new lines at the end are required in most of our projects and when touching file having IDE configured to add them this happens sometimes :) I would prefer to keep it if you don't mind, despite the fact that it's unrelated to PR.

File: build.gradle [hunk 3]
```diff
@@ -118,16 +116,22 @@ subprojects {
...
+            from(project(":storage:core").jar)
+            from(project(":storage:core").configurations.runtimeClasspath)
+            from(project(":storage:s3").jar)
+            from(project(":storage:s3").configurations.runtimeClasspath)
+            from(project(":storage:filesystem").jar)
+            from(project(":storage:filesystem").configurations.runtimeClasspath)
+        }
+    }
+}
+
```
Comment (ivanyu) [inline build.gradle:133]: Checked, this seems to produce identical content with the previous `releaseTarGz`.

**Review by ivanyu — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (AnatolyPopov) [reply build.gradle:133]: Exactly, did the same while was refactoring.

File: docker/kafka/Dockerfile (hunk 2 above)
Comment (AnatolyPopov) [reply docker/kafka/Dockerfile]: Hm, weird.

**Pull request merged**